cli/command/environment.go

```diff
@@ -217,42 +231,52 @@ func (e *EnvironmentCommand) update(c *cli.Context) error {
 }
 
 func (e *EnvironmentCommand) link(c *cli.Context) error {
```

Comment:
this is really hard to read - i'll try to comb through this but will need to come back to it later

Reply:
I think it is an improvement over what was there during WIP stage, but always open to iterate on it to make it more readable.